res/values-pt-rBR/strings.xml

```diff
@@ -1397,20 +1397,20 @@ Comandos:
     <string name="SUDDEN_ATTACK">ATAQUE REPENTINO</string>
 
     <string name="sign_in_explanation" translatable="true">"
-Faça login para acessar o modo multijogador. Você poderá:
+Faça login para acessar o modo multijogador. Assim, você poderá:
 
-- Jogar e se comunicar com outros jogadores de todo o mundo
-- Compartilhar seu progresso com outros
-- Criar e decorar um perfil
-- Comprar plasma na loja
-- Enviar skins criadas pelos usuários
+- Jogar e se comunicar com jogadores de todo o mundo;
+- Compartilhar seu progresso com outros;
+- Criar e personalizar seu perfil;
+- Comprar plasma na loja;
+- Enviar skins criadas por usuários.
 
 Se você não desejar fazer login, ainda poderá jogar no modo único jogador."
     </string>
     <string name="_2x_skin_resolution">Qualidade da Skin em Dobro</string>
     <string name="pm">PRIV.</string>
-    <string name="You_are_already_friends_">Você já tem esta pessoa adicionada.</string>
-    <string name="Account_must_be_1_day_old_to_send_mail_">Esta conta deve ter pelo menos 1 dia de criação para enviar mensagens.</string>
+    <string name="You_are_already_friends_">Você já tem essa pessoa adicionada.</string>
+    <string name="Account_must_be_1_day_old_to_send_mail_">A conta deve ter pelo menos 1 dia de criação para enviar mensagens.</string>
```

Comment:
Line 1412: Previous text was already correct.
Line 1413: "Esta" was already correct for better interpretation.